src/main/java/entities/Item.java

```diff
@@ -1,25 +1,46 @@
 package entities;
 
+import java.text.DateFormat;
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
 import java.util.*;
 
```

Comment:
Consider adding a javadoc/ make a reference to the readme document to denote what attributes of item should be expected